chapters/inheritance.tex

```diff
@@ -564,6 +564,78 @@ \subsection{Final Element Modification Prevention}\label{final-element-modificat
 \end{lstlisting}
 \end{example}
 
+\subsection{Removing Modifiers -- break}\label{removing-modifiers-break}
+Modifications may contain the special keyword \lstinline!break! instead of an expression.
+The intention of \lstinline!break! is to remove the value.
+
+The modifiers using \lstinline!break! are merged using the same rule as other modifications, and follow the same restrictions so they cannot override a final modifier.
+During flattening of an instantiated model, remaining \lstinline!break! modifications (i.e., the ones that are not further overriden) are treated as if the expression was missing.
+The \lstinline!break! modifier for a variable of a simple type can be applied to the value and/or to specific attributes.
+It is possible to override even if no value is present, either because there was no expression originally or because \lstinline!break! overrides another \lstinline!break!.
+
+\begin{nonnormative}
+In a dialog, a tool may hide the keyword \lstinline!break! and show an empty input field, without the overriden modification.
+It should also be possible to remove this modifier to restore the overriden modification.
+
+There are also other uses of the keyword \lstinline!break!, but importantly it is not an expression and thus it cannot be used as a sub-expression.
+\end{nonnormative}
+
+\begin{example}
+Remove unwanted defaults for parameters:
+\begin{lstlisting}
+partial model PartialStraightPipe
+  parameter Real roughness = 2.5e-5 "Average height of surface asperities";
+  parameter Real height_ab (unit = "m" ) = 0 "Height between a and b";
+  ...
+end PartialStraightPipe;
+
+model StaticPipe
+  extends PartialStraightPipe;
+  parameter Real p_a_start = system.p_start;
+  ...
+end StaticPipe;
+
+model MyPipe "Without defaults"
+  extends StaticPipe(
+    p_a_start = break,
+    roughness = break,
+    height_ab = break);
+end MyPipe;
+\end{lstlisting}
+Replace a given parameter value by an initial computation:
+\begin{lstlisting}
+model A
+  parameter Real diameter  = 1;
+  final parameter radius = diameter / 2;
+end A;
+
+model B "Initial equation for diameter"
+  extends A( diameter(fixed = false) = break );
+  parameter Real square=2;
+initial equation
+  //  solving equation below for diameter
+  square = f(diameter);
```

Comment:
If possible using description strings instead of "free floating" comments (which can get lost in the position by the tools reordering) is normally the better style. I would suggest to also make use of this in the examples:
```suggestion
  square = f(diameter) "solving for diameter";
```

Reply:
I don't agree in this case, as _solving for diameter_ isn't what I'd call a _description_ of the equation; it's a comment about how the equation is expected to be solved.  Therefore, a comment makes more sense than a description string.

Reply:
OK fair enough. as for your second question. I'm not sure. Never looked through the specification examples from that perspective. I just know that single line comments are highly unreliable when refactoring models and hence try to avoid them whenever possible.

Reply:
> I just know that single line comments are highly unreliable when refactoring models and hence try to avoid them whenever possible.

Yeah, I know what you mean.

Reply:
I guess this can be marked resolved and the description string vs comments (bike-shed) discussion can be taken up in a different ticket for future improvements.